### PR TITLE
Add events for hrmp pallet.

### LINKED ISSUE #2472: Add events for hrmp pallet
Some events will useful to monitor requests and diagnostic issues.

### PR DESCRIPTION
**Pull request by shaunxw**
Closes #2472

### CODE REVIEW

**Review by pepyakin — APPROVED** [review]
Looks good!

One nit though is that some of the lines started to exceed the soft limit width limit more than I'd like.

File: runtime/test-runtime/src/lib.rs [hunk 1]
```diff
@@ -533,6 +534,7 @@ construct_runtime! {
 		Scheduler: parachains_scheduler::{Module, Call, Storage},
 		ParasSudoWrapper: paras_sudo_wrapper::{Module, Call},
 		SessionInfo: parachains_session_info::{Module, Call, Storage},
+		Hrmp: parachains_hrmp::{Event},
```
Comment (shawntabrizi) [inline runtime/test-runtime/src/lib.rs:537]:
```suggestion
		Hrmp: parachains_hrmp::{Module, Call, Storage, Event},
```

File: runtime/common/src/paras_registrar.rs [hunk 2]
```diff
@@ -290,6 +290,7 @@ mod tests {
 	 		Staking: pallet_staking::{Module, Call, Config<T>, Storage, Event<T>, ValidateUnsigned},
 			Session: pallet_session::{Module, Call, Storage, Event, Config<T>},
 			Initializer: initializer::{Module, Call, Storage},
+			Hrmp: hrmp::{Event},
```
Comment (shawntabrizi) [inline runtime/common/src/paras_registrar.rs:293]:
```suggestion
			Hrmp: hrmp::{Module, Call, Storage, Event},
```

**Review by shawntabrizi — APPROVED** [review]

**Comment by shaunxw** [comment]
Reformatted some long lines to meet the width soft limit. @pepyakin

**Comment by shawntabrizi** [comment]
bot merge

**Pull request merged**